Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.7.md

```diff
@@ -0,0 +1,59 @@
+<h1 align="center" style='color:mediumseagreen;font-weight:bold'>Velaptor Preview Release Notes - v1.0.0-preview.7</h1>
+
+<h2 align="center" style='font-weight:bold'>Quick Reminder</h2>
+
+As with all software, including releases, there is always a chance for issues and bugs.  It is also common to miss changes in the release notes when there are many.  This is even more common in preview releases.
```

Comment:
Put periods on the end of all sentences. 😘

Reply:
😚